vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/payment_util.cc

```diff
@@ -15,20 +18,24 @@ const char kDevelopment[] = "https://payment.rewards.brave.software";
 const char kStaging[] = "https://payment.rewards.bravesoftware.com";
 const char kProduction[] = "https://payment.rewards.brave.com";
 
-std::string GetServerUrl(const std::string& path) {
+std::string GetServerUrl(LedgerImpl* ledger, const std::string& path) {
```

Comment:
this method should not take LedgerImpl just to get kPaymentServiceURL, but this whole `GetStringOption` thing needs to go away. If you want to override the url in the command line just check the command line in the ledger process.

Reply:
or use the existing methods that call `SetEnvironment`